scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1346,5 +1378,5 @@
        (resource-offers [this driver offers]
                         (receive-offers offers-chan driver offers))
        (status-update [this driver status]
-                      (future (handle-status-update conn driver fenzo status))))
+                      (handle-status-update conn driver fenzo status)))
```

Comment:
@wyegelwel How important is that the `status-update` processing returns quickly? I checked the metrics, and it takes less than 1 ms except rare spikes around 1ms (I noticed three spikes on the dashboard). If we really need to do it asynchronously, then I'll need to introduce an agent to ensure the in order processing.